src/components/card.js

```diff
@@ -1,27 +1,22 @@
 import React from "react";
 import "./card.css";
 import { Card as BsCard } from "react-bootstrap/";
-import { SlLogin } from "react-icons/sl";
-
-function Card({ title, content, label }) {
-  const getArrowDirection = () => {
-    return content >= 150 ? "up" : "down";
-  };
 
+function Card({ title, content, label, isPositive, icon }) {
   const getArrowColor = () => {
-    return getArrowDirection() === "up" ? "#009E0F" : "#CC0000";
+    return isPositive ? "#009E0F" : "#CC0000";
   };
 
   return (
     <BsCard.Body className="cardArea">
-      <BsCard.Title className="cardTitle d-flex justify-content-between p-2">
+      <BsCard.Title className="cardTitle  p-2">
         {title}
-        <SlLogin className="icon" />
+        <div className="icon">{icon}</div>
       </BsCard.Title>
       <BsCard.Text className="cardContent p-2">{content}</BsCard.Text>
       <BsCard.Text className="arrowWrap">
```

Comment:
consider conditionally rendering this and grouping all the `value` (10 or 20, this should come from home), `isPositive` and `label` to a single object for example `footer` then only render this when footer is passed

```
{
   footer && (
           <BsCard.Text className="arrowWrap">
           .... other code here
          </BsCard.Text>
      )
}
```